test/IntegrationTests/SqlClientTests.cs

```diff
@@ -55,12 +55,7 @@ public void SubmitTraces()
 
                 foreach (var span in spans)
                 {
-                    span.Service.Should().Be(ServiceName);
-                    span.Name.Should().Be("master");
-                    span.Tags.Should().Contain(new KeyValuePair<string, string>("db.system", "mssql"));
-                    span.Tags.Should().Contain(new KeyValuePair<string, string>("db.name", "master"));
-                    span.Tags.Should().Contain(new KeyValuePair<string, string>("peer.service", $"localhost,{_sqlServerFixture.Port}"));
```

Comment:
Was there a reason for removing these? These seem like good tests to have to ensure the spans have the right information

Reply:
I do not think that it is AutoInstrumentation responsibility to test it.
It should be covered by library tests. Here we should only make a sanity check that instrumentation is _somehow working_.,